specification/storagesync/resource-manager/Microsoft.StorageSync/preview/2020-09-01/examples/RegisteredServers_ListByStorageSyncService.json

```diff
@@ -17,6 +17,8 @@
             "name": "SampleRegisteredServer_1.redmond.corp.microsoft.com",
             "properties": {
               "agentVersion": "3.1.5.0",
+              "agentVersionStatus": "Ok",
+              "agentVersionExpirationDate": "2018-06-11T21:50:40.5840543Z",
               "serverOSVersion": "10.0.14393.0",
               "serverManagementErrorCode": 0,
               "lastHeartBeat": "\"2018-06-11T21:50:40.5840543Z\"",
```

Comment:
Since  lastHeartBeat format change to date-time. so I think \" in string is not required which will cause MV failure

Reply:
Thanks, I didn't realize that was the issue.